_articles/live-disk.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: article
-title: Create and Use Bootable Media from Other OS's
+title: Live Disk Creation (Other Operating Systems)
 description: >
-  You can run Pop!_OS from a USB drive for hardware testing, recovery, and installation/re-installation.
+  Create and Use Bootable on Other Operating Systems.
```

Comment:
Looks like you dropped a word here? ("Media" is still in the other description, but not this one. "Use bootable on" doesn't make sense.)

Also, you might want to get rid of the title capitalization in the new descriptions. None of the other article descriptions are capitalized this way.